AFQ/api.py

```diff
@@ -800,7 +808,7 @@ def _mapping(self, row):
         return mapping_file
 
     def _streamlines(self, row):
-        odf_model = self.tracking_params["odf_model"]
+        odf_model = self.tracking_params["odf_model"].upper()
```

Comment:
Maybe we should make `tractography.track` robust to different input cases?

Reply:
It is. This is for these checks a few lines below:
```
            if odf_model == "DTI":
                params_file = self._dti(row)
            elif odf_model == "CSD":
                params_file = self._csd(row)
            elif odf_model == "DKI":
                params_file = self._dki(row)

```